modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/EpisodeDao.kt

```diff
@@ -338,7 +338,9 @@ abstract class EpisodeDao {
 
     @Query(
         """
-        SELECT DISTINCT podcast_episodes.uuid as episodeId, COUNT(DISTINCT podcast_id) as numberOfPodcasts, SUM(played_up_to) as totalPlayedTime, 
+        SELECT DISTINCT podcast_episodes.uuid as episodeId, COUNT(DISTINCT podcast_id) as numberOfPodcasts,
```

Comment:
nitpick, but I think it would help readability to put the `episodeId` and `numberOfPodcasts` on separate lines like we have with the other parameters. Same with the `mostListenedPodcastId` and `mostListenedPodcastTintColor` on line 345. 

Certainly not a blocker.

Reply:
Good point, updated in https://github.com/Automattic/pocket-casts-android/pull/1482/commits/5a0363c9b22fc8ad491ee40c47befe33dfb0242c